dist/preview release/what's new.md

```diff
@@ -11,6 +11,7 @@
   - Added customShaderNameResolve to PBRMaterialBase to allow subclasses to specify custom shader information [MackeyK24](https://github.com/mackeyk24))
   - Added PBRCustomMaterial to material library to allow easy subclassing of PBR materials [MackeyK24](https://github.com/mackeyk24))  
   - Added AmmoJSPlugin scene file loader [MackeyK24](https://github.com/mackeyk24))  
+  - Added LoadScriptAsync tools helper function [MackeyK24](https://github.com/mackeyk24))  
```

Comment:
This does not seem the right line. you are in the middle of the ammo plugin lines ;)

Probably below in the engine updates